src/content/blog/chrome-built-in-ai-from-nodejs.md

```
@@ -0,0 +1,100 @@
+---
+title: 'Chrome Built-in AI from Node.js'
+subtitle: 'How we run Gemini Nano on a GPU-less headless fleet'
+description: 'Chrome Built-in AI only exists inside a page, and Chrome for Testing will not download Gemini Nano. How we pack the model once, unpack it on the server, and call Prompt, Summarizer, and Language Detector from Node.js — on CPU, with no GPU and no extra API key.'
+date: '2026-08-21'
+---
+
+![](/images/google-gemini-nano-ai-1024x538.jpg)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)